docs/proof-trace.md

```diff
@@ -42,7 +43,9 @@ hook        ::= WORD(0xAA) name arg* WORD(0xBB) kore_term
 ordinal     ::= uint64
 arity       ::= uint64
 variable    ::= name kore_term WORD(0xCC)
-rule        ::= ordinal arity variable*
+rule        ::= WORD(0x22) ordinal arity variable*
+
+side_cond   ::= WORD(0xEE) ordinal arity variable*
```

Comment:
I confirm that this matches the behaviour of the parser, and required minimal code changes on my end to get it working.

Reply:
I added `WORD(0x22)` to the rule event for debugging purposes, I can remove it if you think it's unnecessary.

Reply:
I actually prefer it that way for consistency; we might be able to slim the format down a bit at some point but let's think about that once we have CI and downstream testing set up.